ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+### Problem
+
+### Reproducable Code Sample
```

Comment:
I don't think this is really the right structure for bugs on the notebook repo, because many (most?) issues are about UI, and don't necessarily involve a code sample. We've been happy enough for years without an issue template, so I don't think we need to rush to create one - it's more important to get something that helps more people than it obstructs.